playground/protocol-kit/deploy-safe.ts

```diff
@@ -22,7 +27,7 @@ interface Config {
 
 const config: Config = {
   RPC_URL: sepolia.rpcUrls.default.http[0],
-  DEPLOYER_ADDRESS_PRIVATE_KEY: '<DEPLOYER_ADDRESS_PRIVATE_KEY>',
+  DEPLOYER_ADDRESS_PRIVATE_KEY: SIGNER_ADDRESS_PRIVATE_KEY!,
```

Comment:
Why this comming from the dotenv while the other playgrounds don't. I will address this in a separate PR if it's necessary and align all playgrounds

Reply:
done!